src/restore.rs

```diff
@@ -226,6 +251,31 @@ impl RestoreTree {
                 source,
             }
         })?;
+        #[cfg(unix)]
+        {
+            // Restore permissions
+            let umode = source_entry.umode();
+            fs::set_permissions(&path, umode.into())?;
+            // Restore ownership
+            let owner = source_entry.owner();
+            let uid = if let Some(username) = owner.user {
+                users::get_user_by_name(&username).map(|user| user.uid())
+            } else {
+                None
+            };
+            let gid = if let Some(groupname) = owner.group {
+                users::get_group_by_name(&groupname).map(|group| group.gid())
+            } else {
+                None
+            };
+            // TODO: use `std::os::unix::fs::chown(path, uid, gid)?;` once stable
+            unistd::chown(
```

Comment:
Maybe this should not be fatal? It seems like restoring on a different machine could easily fail to restore users/perms but it should not prevent getting the data back.

Reply:
Good point.